openedx/core/djangoapps/schedules/message_types.py

```diff
@@ -24,3 +28,7 @@ class UpgradeReminder(ScheduleMessageType):
 
 class CourseUpdate(ScheduleMessageType):
     pass
+
+
+class InstructorLedCourseUpdate(ScheduleMessageType):
```

Comment:
Is there any special reason to create a separate object for Instructor paced course updates?

Reply:
yes we want to switch to a separate template for instructor led courses
https://github.com/edx/edx-platform/blob/a2aacc2e935e036adffa9cfb34b205e692d7154e/openedx/core/djangoapps/schedules/resolvers.py#L346

Reply:
Yes, but there isn't any special in this class itself. Both `CourseUpdate` & `InstructorLedCourseUpdate` have same content. Additionally, we are making this feature pacing independent, so unless there is a scenario, this seems unnecessary.

Reply:
basically it inherits from ScheduleMessageType which picks the html based on class name. In this case `InstructorLedCourseUpdate`  template is here openedx/core/djangoapps/schedules/templates/schedules/edx_ace/instructorledcourseupdate/email/body.html

Reply:
ok. It makes sense. Just being curious: why aren't we using the sample template(for self-paced courses) in this case? From what I remember, the email schedule isn't mapped with the course content release date(correct me If I am wrong).

Reply:
It was requirement given by course teams that they wanted different wording and hence a separate template. More details here: https://openedx.atlassian.net/browse/PROD-575

Reply:
Ok. Please add this in concise words in the PR description as not everyone has access to PROD tickets. It will also be easy for the reviewers to understand the decisions made.

Reply:
A sample screenshot of an e-mail would be a :+1: